app/services/availability_service.py

```
@@ -75,20 +76,103 @@ def generate_time_slots(self, start_str: str, end_str: str) -> List[str]:
         start_hour와 end_hour 사이의 1시간 단위 슬롯 리스트를 생성합니다.
         예: 14:00 ~ 16:00 -> ["14:00", "15:00", "16:00"]
         """
-        start_time = datetime.strptime(start_str, "%H:%M")
-        end_time = datetime.strptime(end_str, "%H:%M")
-        
-        if start_time > end_time:
+        start_min = self._slot_to_minutes(start_str)
+        end_min = self._slot_to_minutes(end_str)
+
+        if start_min > end_min:
             raise ValueError("시작 시간이 종료 시간보다 같거나 늦을 수 없습니다.")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)